{
  try {
    connection.abort();
  }
 catch (  Exception e) {
    log.error(e,"Connector threw exception on abort");
  }
}
